src/output.c

```diff
@@ -271,8 +271,9 @@ outputproc(struct outargs *oa)
 	}
 
 end:
-	/* this is breaking the format, but still useful information */
-	fprintf(evout, "{\"duration\":%0.3f}", dur / 1000);
+	if (oa->append_trailer) {
```

Comment:
Per the v2 format, duration is a thing that is allowed in the header. If we're going to print it out anywhere, I'd like for it to be there rather than as an optional thing that is outside the spec.